src/main/resources/db/migration/postgres/V1_320_1__copyCaseNumbersToMediaLinkedCase.sql

```diff
@@ -0,0 +1,20 @@
+DO
+$do$
+DECLARE
+   case_id   int;
+   courthouse_name   text;
+	temprow record;
+BEGIN
+FOR temprow IN
+        select med_id, ctr_id, UNNEST(case_number) case_number from darts.media where case_number is not null order by 1,2
```

Comment:
Although not related to this pr. Should the table definition have a mutual exclusive check constraint:-

CHECK ((case_number IS NULL AND cas_id IS NOT NULL) 
   OR (cas_id IS NOT NULL AND case_number IS NULL))

Reply:
One for @barry-jones50410